File: security/src/main/java/com/ftn/security/authentication/model/AuthenticationResponse.java
package com.ftn.security.authentication.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@RequiredArgsConstructor
@AllArgsConstructor
public class AuthenticationResponse {

    private String jwt;
    private String mail;
    private String role;

}

File: SecurityFrontendApp/src/app/model/AuthenticationResponse.ts
export class AuthenticationResponse {

    jwt: string;
    mail: string;
    role: string;

    constructor(jwt = "", mail = "", role = "") {
        this.jwt = jwt;
        this.mail = mail;
        this.role = role;
    }

    public isAdmin() {
        return this.role === 'ADMIN';
    }

    public isUser() {
        return this.role === 'USER';
    }
   
}